profiles/common/modules/custom/nexteuropa_dgt_connector/tmgmt_poetry/plugins/tmgmt_poetry.plugin.inc

```diff
@@ -257,10 +257,16 @@ class TMGMTPoetryTranslatorPluginController extends TMGMTDefaultTranslatorPlugin
     // Generate the callback parameters.
     $callback_username = $settings['callback_user'];
     $callback_password = $settings['callback_password'];
-    $callback_address = url(drupal_get_path("module", "tmgmt_poetry") . "/wsdl/PoetryIntegration.wsdl", array(
-      'absolute' => TRUE,
-      'language' => (object) array('language' => FALSE),
-    ));
+    $poetry = variable_get("poetry_service");
+    if (isset($poetry['callback_address'])) {
+      $callback_address = $poetry['callback_address'];
```

Comment:
@enriquelacoma  please align with @Tiago-FM  to make sure we implement the same as D8.
If they have to set up "PATH/wsdl/PoetryInregration.wsdl" for D7 and "PATH/poetry/notifications.wsdl" for D8 that's more confusing.
Ideally they should only set up "PATH"

tx